docs/source/links.rst

```diff
@@ -37,6 +36,37 @@
 .. _IR Fall-out: https://en.wikipedia.org/wiki/Evaluation_measures_(information_retrieval)#Fall-out
 .. _MAPE implementation returns: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_absolute_percentage_error.html
 .. _mean squared logarithmic error: https://scikit-learn.org/stable/modules/model_evaluation.html#mean-squared-log-error
-.. _Mean Reciprocal Rank: https://en.wikipedia.org/wiki/Mean_reciprocal_rank
 .. _LPIPS: https://arxiv.org/abs/1801.03924
 .. _Tweedie Deviance Score: https://en.wikipedia.org/wiki/Tweedie_distribution#The_Tweedie_deviance
+.. _Permutation Invariant Training of Deep Models: https://ieeexplore.ieee.org/document/7952154
```

Comment:
I may suggest adding one by one where each commit will be moving one link here to this aggregating list and replacing it accordingly in the codebase so it is much easier to find which changes is breaking it...